Tools/ardupilotwaf/toolchain.py

```diff
@@ -126,13 +126,17 @@ def find_toolchain_program(cfg, filename, **kw):
     return cfg.find_program(filename, **kw)
 
 def configure(cfg):
+    _filter_supported_c_compilers('gcc', 'clang')
+    _filter_supported_cxx_compilers('g++', 'clang++')
+    
     if cfg.env.TOOLCHAIN == 'native':
         cfg.load('compiler_cxx compiler_c')
+        if cfg.env.COMPILER_CC == 'clang' or cfg.env.COMPILER_CXX == 'clang++':
+            if cfg.env.DEST_OS == 'darwin':
```

Comment:
No need to to check for macOS, clang isn't being looked after at this point, giving the warning in all systems is good.
Also, we should put the warning for all builds, so before the _native_ check.

Reply:
ok